vending-app/src/main/kotlin/com/google/android/finsky/DeviceSyncInfo.kt

```diff
@@ -0,0 +1,612 @@
+/**
+ * SPDX-FileCopyrightText: 2024 microG Project Team
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.google.android.finsky
+
+import android.accounts.Account
+import android.accounts.AccountManager
+import android.annotation.SuppressLint
+import android.app.ActivityManager
+import android.app.admin.DevicePolicyManager
+import android.content.Context
+import android.content.pm.PackageInfo
+import android.content.pm.PackageManager
+import android.content.res.Configuration
+import android.graphics.Point
+import android.opengl.GLES10
+import android.os.Build
```

Comment:
Instead of using real device data, can you please use data from the ProfileManager where possible?

Reply:
Modified